GW2EIEvtcParser/EIData/ProfHelpers/Mesmer/MirageHelper.cs

```diff
@@ -14,7 +14,8 @@ internal static class MirageHelper
 
         internal static readonly List<InstantCastFinder> InstantCastFinder = new List<InstantCastFinder>()
         {
-            new DamageCastFinder(Jaunt, Jaunt), // Jaunt
+            new DamageCastFinder(Jaunt, Jaunt).UsingEnable(combatData => !combatData.HasEffectData),
+            new EffectCastFinderByDst(Jaunt, EffectGUIDs.MirageJaunt).UsingDstSpecChecker(Spec.Mirage),
```

Comment:
Are you sure about that? I remember having to remove Jaunt effect finder as the 3 effect IDs I had identified were ultimately colliding with something else.

Reply:
I didn't manage to find a conflict with the skills I tested manually (Staff 2, Axe 3, Blink, Illusionary Ambush). It's possible it collides with something I did not test. Do you have any idea what it could've collided with?

Reply:
Unfortunately I don't remember as that was nearly a year ago, however Jaunt is in my "checked" list and marked as all IDs colliding. The tested golem log contains the effect IDs you have added. It may be worth double checking however.